DUMMY_PROCESS_TO_BE_REMOVED.md

```diff
@@ -0,0 +1,20 @@
+
+this is a file to be used for documenting processes, this will be removed shortly.
+
+```python
+def rick_roll():
+    print("We're no strangers to love")
+    print(" You know the rules and so do I")
+    print(" A full commitment's what I'm thinking of")
+    print(" You wouldn't get this from any other guy")
+
+    print(I just wanna tell you how I'm feeling")
+    print("Gotta make you understand")
+
+    print("Never gonna give you up")
```

Comment:
Very good song!

Reply:
I agree! To bad you don't know the rick roll :(